jmeter/plugins/jmeter-backendlistener-maestro/README.md

```diff
@@ -0,0 +1,63 @@
+Maestro Backend Listener for JMeter
+----
+Maestro Backend Listener is a JMeter plugins that sends sample results to Maestro API.
+
+
+## Sample Result Metrics
```

Comment:
I think we could put a link to swagger docs in the API to make sure we have everything updated, even if anything changed